solr/core/src/java/org/apache/solr/handler/admin/api/NodeLoggingAPI.java

```diff
@@ -17,19 +17,34 @@
 
 package org.apache.solr.handler.admin.api;
 
-import static org.apache.solr.client.solrj.SolrRequest.METHOD.GET;
-import static org.apache.solr.security.PermissionNameProvider.Name.CONFIG_EDIT_PERM;
-
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.enums.ParameterIn;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.info.License;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.parameters.RequestBody;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
 import org.apache.solr.api.EndPoint;
+import org.apache.solr.client.solrj.request.beans.MoveReplicaPayload;
 import org.apache.solr.handler.admin.LoggingHandler;
 import org.apache.solr.request.SolrQueryRequest;
 import org.apache.solr.response.SolrQueryResponse;
 
+import javax.ws.rs.Path;
+
+import static org.apache.solr.client.solrj.SolrRequest.METHOD.GET;
+import static org.apache.solr.security.PermissionNameProvider.Name.CONFIG_EDIT_PERM;
+
 /**
  * V2 API for getting or setting log levels on an individual node.
  *
  * <p>This API (GET /v2/node/logging) is analogous to the v1 /admin/info/logging.
  */
+@OpenAPIDefinition(info = @Info(title = "Solr v2 API", description = "Some description", license = @License(name = "ASL 2.0"), version = "0.0.1"))
```

Comment:
Is it possible to set global values for some of these, like we know everything is going to be ASLv2 Licenses and maybe we tie version to our release version? Or is that supposed to be an API version that is more stable than our releases.